examples/aws/ingress/values.yaml

```diff
@@ -1,14 +1,21 @@
 # This example shows how to deploy and expose GraphDB with AWS Load Balancer Controller Ingress without SSL.
 
 configuration:
-  externalUrl: http://graphdb-example-dns-name.com/ # Change this to your ALB DNS name or Route53 if you use it.
+  # Change this to your AzureDNS domain name or use nip.io which you can do after you deploy the chart since you
+  # need to map the public IP address of the Application Gateway to the nip.io.
+  externalUrl: http://ip.address.nip.io/
```

Comment:
Shouldn't it be possible to use the IP address from the assigned public IP, e.g. `http://<public-ip>`? I think that `nip.io` is not really needed. But you could leave it as an example.

Reply:
No, I've tried that before committing and it returns an error that it cannot be an IP, but FQDN or something like that.

Reply:
The exact error Error: INSTALLATION FAILED: 1 error occurred: helm install graphdb ontotext/graphdb -f examples/ingress-nginx/values.yaml                                                                                                                                                     ✔  at kind-blaa ⎈  at 13:13:55  
        * Ingress.networking.k8s.io "graphdb" is invalid: spec.rules[0].host: Invalid value: "127.0.0.1": must be a DNS name, not an IP address

Reply:
Yeah.. ingresses dislike IPs and the property is used to configure the default.